ccd-definition/ComplexTypes/LitigationFriend.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "ID": "LitigationFriend",
+    "ListElementCode": "required",
```

Comment:
can we change this to litigationFriendRequired?

Reply:
No, isn’t needed. Complex type structure gives context as

litigantFriend: {
required: “”
}

Reply:
ok